Makefile

```diff
@@ -1,4 +1,12 @@
-.PHONY: all submodules server client clean purge test help default fb
+.PHONY: all deps migrate server client prepare build clean purge fb help default
```

Comment:
`make submodules` should still be possible, right?

Reply:
hm, there is no `prepare` in the code

Reply:
+1, probably prepare is create, but I think prepare has more sense here.